src/scheduling/dimacs_change.h

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
This file has a misleading name. It only prints a comment. It doesn't generate a change.